modules/repository/create_test.go

```diff
@@ -169,3 +169,13 @@ func TestUpdateRepositoryVisibilityChanged(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, act.IsPrivate)
 }
+
+func TestGetDirectorySize(t *testing.T) {
+	assert.NoError(t, unittest.PrepareTestDatabase())
+	repo, err := repo_model.GetRepositoryByID(db.DefaultContext, 1)
```

Comment:
```suggestion
	repo, err := repo_model.GetRepositoryByID(1)
```
This should appease the linter.